README.md

```diff
@@ -19,6 +19,8 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 
 Tested with the current Ansible 2.9, ansible-base 2.10, ansible-core 2.11, ansible-core 2.12 releases and the current development version of ansible-core. Ansible versions before 2.9.10 are not supported.
 
+Please note that some content will not work with Ansible 2.9 releases before 2.9.20, and ansible-base 2.10 releases before 2.10.8. If you are using such a version, please upgrade to a newer (or better: latest) version of Ansible 2.9 respectively ansible-base 2.10, or preferably to an ansible-core version.
```

Comment:
```suggestion
Please note that some content will not work with Ansible 2.9 releases before 2.9.20, and ansible-base 2.10 releases before 2.10.8. If you are using such a version, please upgrade to a newer (or better: latest) version of Ansible 2.9 or ansible-base 2.10 respectively, or preferably to an ansible-core version.
```
Maybe? if this wording is intended, feel free to just marked this comment as resolved.